### DISCUSSION TITLE
Feature: Team Page

### DISCUSSION THREAD

**Discussion by GrantBirki** [arthelokyo/astrowind#276] [Ideas]
# Team Page

Hey there! 👋 

To start off here is some context, a while back a created my own fork of your project. I wanted to add a few features to my fork and ended up making quite a few modifications. Turns out, you folks and the folks over at Astro move **really freaking fast**. I straight up cannot keep up with the awesome work that you all have been doing. Anyways, I am abandoning my fork and want to contribute changes here instead!

I am by no means a web developer so take my suggestions with a grain of salt.

Here is my first one, a _team page_

## Example 📸 

![Image](https://github.com/onwidget/astrowind/assets/23362539/bb519d77-02bf-4aa2-a5d3-e5a622684b8d)

I created this `/team` page with a `.jsx` component in Astro. The source code for this component that I make [can be found here](https://github.com/GrantBirki/astrowind-hard-fork/blob/be29d05cc0f3fe04e37ade9d38c653ed55c6cf53/src/components/widgets/Team.jsx) and [here](https://github.com/GrantBirki/astrowind-hard-fork/blob/be29d05cc0f3fe04e37ade9d38c653ed55c6cf53/src/pages/team.astro) for reference.

## Live Demo

A live demo of my site where this lives can be found [here](https://astro-demo.birki.io/team/)

---

I think this component would be a great idea, feel free to use mine as inspiration should you chose to create one! ⭐

**Comment by prototypa** [comment]
Hi @GrantBirki 

Thank you very much for the contribution. You may have noticed that I have moved the Issue to Discussion/Ideas since you are not reporting an issue with **AstroWind**.

I was checking out your _Team Widgets_. It is useful. There are certainly some changes that we would make to integrate it coherently with AstroWind:

- Create a typescript for Team and define common properties that we have for all widgets + the Team Widget properties.

- Use the `WidgetWrapper` component to wrap all the widget content (ex: https://github.com/onwidget/astrowind/blob/77ab3f63d5b092f4f6b3a4a6e660f7d17b332eb8/src/components/widgets/Features.astro#L23C15-L23C15)

- Use `Headlines` for the title and header content of the Widgets (ex: https://github.com/onwidget/astrowind/blob/77ab3f63d5b092f4f6b3a4a6e660f7d17b332eb8/src/components/widgets/Features.astro#L29)

- Use existing icons in tabler icons through the _astro-icon_ dependency instead of the `svg` you put there.

- Have a mapping of social network vs icons

- Optimize the images with the Images component that we have in **AstroWind** (ex: https://github.com/onwidget/astrowind/blob/main/src/components/widgets/Content.astro#L78)

And maybe other details.

I find it very convenient that instead of me making the Team Widget based on what you have shown, it is better for everyone that you make a Pull Request and contribute code to **AstroWind** from now on. In the PR you could adjust some of the details that I suggest, as many as you can, and we would do the rest.

Greetings

**Comment by abusayed0206** [comment]
Hey, appreciate this project and the 'team page'

Any update on the team page? 

@GrantBirki 
@prototypa
